### DISCUSSION TITLE
Relation between `gunicorn` and `uvicorn.workers.UvicornWorker` workers

### DISCUSSION THREAD

**Discussion by carlbordum** [Kludex/uvicorn#2013] [Q&A] [answered]
When running `gunicorn -k uvicorn.workers.UvicornWorker`, you can configure `gunicorn` with `--workers`. Can you simultanously configure uvicorn workers?

What's the relation between the two?

**Comment by Kludex** [comment, marked as answer]
I'm not sure if I get the question, but if you use gunicorn, is gunicorn who creates the uvicorn processes. If you use the `workers` flag from uvicorn, it will be ignored.

**Reply by carlbordum** [reply]
Thanks. _It will not be used_ is good to know :)
